pkg/setting/setting.go

```diff
@@ -1263,6 +1264,7 @@ func (cfg *Cfg) readSAMLConfig() {
 	samlSec := cfg.Raw.Section("auth.saml")
 	cfg.SAMLAuthEnabled = samlSec.Key("enabled").MustBool(false)
 	cfg.SAMLSkipOrgRoleSync = samlSec.Key("skip_org_role_sync").MustBool(false)
+	cfg.SAMLRoleValuesGrafanaAdmin = samlSec.Key("role_values_grafana_admin").MustString("")
```

Comment:
If we wanted to be exhaustive, we could also check that the `role_values_grafana_admin` formatted correctly, but I think it's a bit over the top.